apps/web/src/components/Onboarding/ImportWallet/SeedPhraseTab.tsx

```diff
@@ -88,10 +91,24 @@ export const SeedPhraseTab = () => {
       range(mnemonicSize).map(() => ({ val: "" }))
     );
 
+  const lastRowSize = useBreakpointValue({ md: fields.length % 4 }) || 0;
+
+  const onPaste: InputProps["onPaste"] = event => {
+    event.preventDefault();
+    void pasteMnemonic(event.clipboardData.getData("text/plain"));
```

Comment:
Why not `return pasteMnemonic(event.clipboardData.getData("text/plain"));`

Reply:
the callback type doesn't allow it :(